### ISSUE TITLE
Frontpage content stops loading after a few pages on Galaxy S7

### ISSUE THREAD

**Issue by zagaberoo** [zagaberoo/diode#90]
A user reports the following issue followed by total failure to load further pages:

![image](https://user-images.githubusercontent.com/1527500/68788808-fe9dbc80-05f8-11ea-833f-18b8c27bf5a0.png)

Under these circumstances:

* signed in
* Android 8
* Galaxy S7
* Diode 1.3.2.2 (26)
* possibly only when not on wifi, though issues are not shared across other apps

**Comment by matevskial** [comment]
Yes, I can confirm that this problem happens under similar circumstances:

* signed in
* Android 9
* Diode 1.3.2.2 (26)

For me, it also happens on Wi-Fi.

Additionally, when this situation happens, I am not able to read comments of a particular post, I have to wait some time, refresh, or restart the app completely.

I suspect there is a problem with loading image thumbnails.
